### DISCUSSION TITLE
Run Installer Twice

### DISCUSSION THREAD

**Discussion by Jayson900** [bleachbit/bleachbit#1369] [Q&A]
Why does BleachBit prompt me to run the install setup twice?

**Comment by abitrolly** [comment]
I am afraid I can not answer this without a screencast to see when and how that happens.

**Comment by az0** [comment]
This gives the user the option to install under the user profile without requiring administrative privileges or installing for all users with admin privileges. When the installer first starts, it doesn't have admin privileges, but when they are required, it escalates it privileges. 

The BleachBit application does something similar. It starts without admin privileges and requests them. If the user declines, it will continue running without admin privileges, but cleaning some functions will not be available. If user accepts, it spawns a second instance (in a less awkward way than the installer).
